src/NzbDrone.Core/Indexers/Definitions/AnimeBytes.cs

```diff
@@ -228,10 +228,18 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                         if (syn.StringArray != null)
                         {
                             synonyms.AddRange(syn.StringArray);
+                            if (syn.StringArray.Count >= 3)
+                            {
+                                synonyms.AddRange(syn.StringArray[2].Split(',').Select(t => t.Trim()));
+                            }
                         }
                         else
                         {
                             synonyms.AddRange(syn.StringMap.Values);
+                            if (syn.StringMap.ContainsKey("2"))
```

Comment:
Same as above, will add the one with key 2 two times instead handling this edge case